djvm/src/main/kotlin/net/corda/djvm/analysis/ClassAndMemberVisitor.kt

```diff
@@ -591,7 +582,7 @@ open class ClassAndMemberVisitor(
         /**
          * The API version of ASM.
          */
-        const val API_VERSION: Int = Opcodes.ASM6
+        const val API_VERSION: Int = Opcodes.ASM7
```

Comment:
For my understanding, what is the difference between ASM6 and ASM7? Where is the API_VERSION relevant?

Reply:
I've updated the ASM library from 6.2.1 to 7.1, and so I've updated the API version accordingly. There was no reason not to...

Basically, the API version allows the ASM library writers to change their API while still being backwards compatible with earlier versions. But the DJVM doesn't need to worry about that.